### ISSUE TITLE
landsat, satellite, RStoolbox examples

### ISSUE THREAD

**Issue by jannes-m** [Robinlovelace/geocompr#107]
Reviewer 1 thinks that examples with landsat, satellite, and/or RStoolbox packages could be helpful in chapter 4 spatial operations. I would suggest to wait because if we use these packages in chapters later on, we can make a cross-reference. If not, I am happy to include such an example in chapter 4.

**Comment by Nowosad** [comment]
@jannes-m What is your thinking about it now?

**Comment by Robinlovelace** [comment]
I don't think we have time or space for examples. I suggest just briefly mentioning them in the main text of c4 will be sufficient to close this issue.

**Comment by jannes-m** [comment]
I agree with @Robinlovelace. We have already mentioned these packages, I've just added a reference to the Remote Sensing R book as a further pointer.

Actually, I thought we had an exercise that uses one the mentioned packages to compute an index but I can't find it no longer...
